cli/crates/server/src/config/mod.rs

```diff
@@ -171,6 +176,14 @@ async fn parse_and_generate_config_from_ts(ts_config_path: &Path) -> Result<Stri
     validate_node().await?;
     let node_command = Command::new("node")
         .args(args)
+        .env(
+            "GRAFBASE_ENV",
+            match environment_name {
+                EnvironmentName::Production => "production",
```

Comment:
missing Preview envs

Reply:
Those don't exist for the moment in the CLI

Reply:
And this specific distinction is only for `dev` vs `prod` but we could add them